tests/Builder/Query/BitsAllClearOperatorTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MongoDB\Tests\Builder\Query;
+
+use MongoDB\BSON\Binary;
+use MongoDB\Builder\Pipeline;
+use MongoDB\Builder\Query;
+use MongoDB\Builder\Stage;
+use MongoDB\Tests\Builder\PipelineTestCase;
+
+/**
+ * Test $bitsAllClear query
+ */
+class BitsAllClearOperatorTest extends PipelineTestCase
+{
+    public function testBinDataBitmask(): void
+    {
+        $pipeline = new Pipeline(
+            Stage::match(
+                a: Query::bitsAllClear(
+                    new Binary('MC=='),
```

Comment:
Per my earlier comment (https://github.com/mongodb/mongo-php-builder/pull/37/files#r1458006767), you'll need to use `base64_decode()` to prepare the binary string for the Binary constructor. Likewise for other test files.

And be sure to use the canonical base64 representations. I'm not sure how PHP handles this, but it'd be preferable not to rely on any behavior that truncates to 8-bit boundaries.